### PR TITLE
#45 adding retransmission dry run mode

### PR DESCRIPTION
**Pull request by druminski**

### CODE REVIEW

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/retransmit/PartitionOffsetChange.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package pl.allegro.tech.hermes.management.infrastructure.kafka.service.retransmit;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PartitionOffsetChange {
```
Comment (adamdubiel) [inline hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/retransmit/PartitionOffsetChange.java]: we already have `PartitionOffset` object, and they don't need to be deserializable (no need for `@JsonCreator`)

Comment (druminski) [inline hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/retransmit/PartitionOffsetChange.java]: good observation. I have removed PartitionOffsetChange class. However I also added @JsonCreator annotation to PatitionOffset class due to deserialization done in integration test for retransmission.

**Comment by pawelszymczyk** [comment]
:+1:

**Comment by adamdubiel** [comment]
:+1:

**Comment by mieciu** [comment]
:cake:

**Pull request merged**